test/DropdownSpec.js

```diff
@@ -333,4 +333,141 @@ describe('<Dropdown>', () => {
       ).assertSingle('div.dropdown-menu');
     });
   });
+
+  describe('autoClose behaviour', () => {
+    const createDocumentListenersMock = () => {
```

Comment:
Do we need this?  We have other methods in other tests to trigger click events.

Reply:
I didn't notice any method concerned with simulating clicks outside of the component. Correct me if I'm wrong. Should we keep the outside click methods?

Reply:
We did use `simulant.fire(document.body, 'click');` inside an Offcanvas test.  Can we try that and see if it works for your test?